vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -577,6 +580,9 @@ func (d *VolumeDriver) Mount(r volume.MountRequest) volume.Response {
 func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Unmounting Volume ")
 
+	// dirty the background refcounting process if any
+	d.refCounts.MarkDirty()
+
 	// if the volume is still used by other containers, just return OK
```

Comment:
if it's getting dirty there is no reason to manipulate refcount, is there ? 
I'd expect somethinng like 
```go
 if d.refCounts.MarkDirty() != nil {
   logWithFields(...) # lines 596-600
}

refcnt, err= d.decrRefCount(r.Name)
...

```